core/translations/i18n/en.json

```diff
@@ -49,82 +49,7 @@
   "Unloading module \"${0}\".": "Unloading module \"${0}\".",
   "What kind of shit platform is this that doesn't even support sending messages?": "What kind of shit platform is this that doesn't even support sending messages?",
   "Working on it...": "Working on it...",
-
```

Comment:
Oh! fair enough, though I ended up changing this to `Concierge module` in the translation PR; before I do another PR for that why don't we change `kassy.json` to `module.json` or `config.json` instead?